### PR TITLE
whoops

### PR DESCRIPTION
**Pull request by makosblade**
Untested

### CODE REVIEW

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationRunner.kt [hunk 1]
```diff
@@ -150,7 +149,7 @@ class IntegrationRunner {
             try {
                 ds.write()
                         .option("batchsize", destination.batchSize.toLong())
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationRunner.kt:151]: is this one "batchsize" or "batchSize" (based on LaunchpadConstant below) ? Or does it not matter?

**Review by katherinebernstein — APPROVED** [review]
Is "launch pad" normally two words? I feel like I usually/always see it as one (and also grammarly is currently confirming that belief by trying to correct what I just typed to `launchpad`) so `LaunchPad` feels wrong, but anyway the code part looks good

**Comment by katherinebernstein** [comment]
> Is "launch pad" normally two words? I feel like I usually/always see it as one (and also grammarly is currently confirming that belief by trying to correct what I just typed to `launchpad`) so `LaunchPad` feels wrong, but anyway the code part looks good

other than the conflicts. Is this PR super old? looks like it 😢

**Comment by makosblade** [comment]
Ohhh lol this is ancient. I'll probably redo this on a bored day off or something later. Closing for now

**Pull request closed without merging**